src/components/menu.svelte

```diff
@@ -22,14 +23,26 @@
 </script>
 
 <div id="menu-component">
-  <button id="open-button" class="open-button" on:click={() => (open = true)}>Open</button>
+  <IconButton
+    id={'open-button'}
+    buttonStyle="position: fixed; z-index: 0; top: 0; left: 0;"
+    iconClass="fa-solid fa-bars"
+    onclick={() => (open = true)}
+  />
   <div id="menu" class="menu {open ? 'open' : ''}">
     <header id="menu-header">
-      {#if menuStackLength > 1}
-        <button id="back-button" on:click={() => (menuStack = pop(menuStack))}>Back</button>
-      {:else}
-        <button id="back-button" on:click={() => (open = false)}>Close</button>
-      {/if}
+      <IconButton
+        id={'back-button'}
+        buttonStyle="position: absolute; top: 0; left: 0;"
```

Comment:
Code looks a little wonky, but I was unable to get the css compiler to keep the `#open-button` style definition by passing `back-button` into the component. I could also include the open and back button styles in menu.css if I wanted, but eh.